app/src/main/java/com/rob729/quiethours/Fragments/MainFragment.kt

```diff
@@ -76,7 +75,7 @@ class MainFragment : Fragment() {
         binding.rv.adapter = profileListAdapter
         binding.rv.layoutManager = LinearLayoutManager(context)
 
-        binding.lifecycleOwner = this
+//        binding.lifecycleOwner = this
```

Comment:
@rob729 this line was showing unresolved reference for "lifecycleOwner ". I checked the app after commenting it out, and it was working fine. 
I was not sure about its usage, so instead of completely removing it, I have commented it out.

Reply:
Remove this line. It is related to data binding,

Reply:
@rob729 Ok

Reply:
Done